packages/components/src/styles/table-cell/table-cell.css

```diff
@@ -0,0 +1,25 @@
+.sd-table-cell {
+  @apply border-solid border-neutral-400 border-y-[1px] border-x-0 text-black text-sm text-left p-4;
+
+  &--divider {
+    @apply border-r-[1px];
+  }
+
+  &--bg {
+    &-transparent {
+      @apply bg-transparent;
+    }
```

Comment:
I think this should be part the default. Please remove it here and in the declaration and add bg-transparent to line 2.